_includes/setup/system-win.md

```diff
@@ -0,0 +1,11 @@
+## System requirements
+
+To install and run Flutter, your development environment must meet these minimum requirements:
+
+* **Operating Systems**: Windows (64-bit)
```

Comment:
Should we be explicit about the Windows version? We don't intent to support any version before Windows 7.